lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -6,10 +6,18 @@ END_JOURNEY:
 
 CRI_STATE:
   events:
+    next:
+      targetState: ERROR
+    not-found:
+      targetState: ERROR
+    fail-with-no-ci:
+      targetState: ERROR
     error:
       targetState: ERROR
     access-denied:
       targetState: PYI_NO_MATCH
+    enhanced-verification:
+      targetState: ERROR
```

Comment:
I think we might want to change some of these defaults - e.g. so the failure cases route to `PYI_NO_MATCH` or `PYI_ANOTHER_WAY`, but in the spirit of separating the tests from functional changes, I'll raise a new ticket.

Reply:
Thank you - I think to avoid potentially breaking a flow, it'd be better done in a dedicated bit of work